test/client-dispatch.js

```diff
@@ -91,7 +91,7 @@ test('basic dispatch get', (t) => {
   })
 })
 
-test('trailers dispatch get', (t) => {
+test('trailers dispatch get', { only: true }, (t) => {
```

Comment:
What does `only: true` do?

Reply:
I forgot to remove it after debugging, it runs only test tests marked with only when using the option `--only` or the env var `TAP_ONLY=1`.